compiler/symbol.go

```diff
@@ -208,6 +208,12 @@ func (c *compilerContext) getFunction(fn *ssa.Function) (llvm.Type, llvm.Value)
 			// > circumstances, and should not be exposed to source languages.
 			llvmutil.AppendToGlobal(c.mod, "llvm.compiler.used", llvmFn)
 		}
+	case "GetModuleHandleExA", "GetProcAddress", "GetSystemInfo", "GetSystemTimeAsFileTime", "LoadLibraryExW", "QueryUnbiasedInterruptTime", "SetEnvironmentVariableA", "Sleep", "VirtualAlloc":
```

Comment:
Can this be done as a catch-all when a kernel32.dll function is being imported? 
Also this fixup might be required for all external dll calls in general. Would need more testing.

Reply:
I suppose it can, by parsing `//sys` comments in the source code. But just adding the list of names was a lot easier.

Programs that would like to call kernel32.dll functions should use the syscall package, which uses the appropriate calling convention.